### ISSUE TITLE
Issues when inputs have brackets "[ ]" in the name attribute

### ISSUE THREAD

**Issue by kbherbert** [AudithSoftworks/Uniform#325]
Currently running Uniform 2.1.1 on a .NET MVC application. Some of my inputs have name attributes that are returned from a web service and when we iterate through them we get something like this:

```
<input checked="checked" id="preferenceID-1" name="PrefereceModel.CommPrefs[0].UserPreferences[0].PreferenceId" type="radio" value="Phone">
```
```
<input checked="checked" id="preferenceID-1" name="PrefereceModel.CommPrefs[0].UserPreferences[0].PreferenceId" type="radio" value="Email">
```

What I am finding is that Uniform does not like the brackets [ ] at all.  In fact, it causes radios with the same name to either all be select-able, or not select-able at all! This breaks my application. Removing the brackets fixes the issue, but I cannot do this due to the dynamic nature of the form.

Note: This issue previously only existed in IE7, but is now present in all browsers with version 2.1.1.

**Comment by shehi** [comment]
@fidian, can't we just set a minimal jQuery version required and be done with it? Is this problem still plaguing the package?

**Comment by fidian** [comment]
That is one solution that would work, though it wouldn't work everywhere.  Some larger companies don't like to be on the "latest" even when their version is several releases out of date.  If that's an acceptable risk, this issue could be closed by requiring 1.8.

**Comment by shehi** [comment]
I will keep it open as this package lacks in testing department big time and resources to get them done, not to mention the very topic it addresses isn't quite popular one - and there are tons of similar packages doing the same thing, partially or otherwise.
